### PR TITLE
Updated DscJobs presence.

### PR DESCRIPTION
**Pull request by DiscordAnaxes**
Proof of taking over ownership:

Proof of presence working:

### CODE REVIEW

File: websites/D/DscJobs/dist/metadata.json [hunk 1]
```diff
@@ -1,15 +1,16 @@
 {
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "DscJobs",
   "author": {
-    "name": "Dexter",
-    "id": "780079091172900884"
```
Comment (ririxidev) [inline websites/D/DscJobs/dist/metadata.json:4]: add him as contributor

File: websites/D/DscJobs/presence.ts [hunk 2]
```diff
@@ -1,94 +1,114 @@
...
-          {
-            url: location.href,
-            label: `Visit ${name || "Dummy"}'s Profile`
-          }
-        ];
-      } else presenceData.details = "Unknown Page";
-      break;
+if (document.location.pathname === "/") {
+  presenceData.details = "Viewing home page";
+} else if (document.location.pathname.includes("/search")) {
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:15]:
```suggestion
if (document.location.pathname === "/") 
  presenceData.details = "Viewing home page";
  else if (document.location.pathname.includes("/search")) {
```

File: websites/D/DscJobs/presence.ts [hunk 3]
```diff
@@ -1,94 +1,114 @@
...
+  ];
+} else if (document.location.pathname === "/moderators") {
+  const moderator_page = document.location.href
+      .split("page=")[1]
+      .split("&")[0],
+    filters = document.location.href.includes("&sort=");
+  presenceData.details = `Viewing 🔨 hireable moderators - Page: ${moderator_page}`;
+  if (filters) {
+    presenceData.state = `💿 Filters: True`;
   }
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:34]:
```suggestion
  if (filters) presenceData.state = `💿 Filters: True`;
```

File: websites/D/DscJobs/presence.ts [hunk 4]
```diff
@@ -1,94 +1,114 @@
...
+      label: "View Results",
+      url: document.location.href
+    }
+  ];
+} else if (document.location.pathname === "/moderators") {
+  const moderator_page = document.location.href
+      .split("page=")[1]
+      .split("&")[0],
+    filters = document.location.href.includes("&sort=");
+  presenceData.details = `Viewing 🔨 hireable moderators - Page: ${moderator_page}`;
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:31]: maybe this page number will be better in state?

File: websites/D/DscJobs/presence.ts [hunk 5]
```diff
@@ -1,94 +1,114 @@
...
+  presenceData.details = "Viewing 🤝 partners";
+  presenceData.buttons = [
+    {
+      label: "View Partners",
+      url: document.location.href
+    }
+  ];
+} else if (document.location.pathname.includes("/profile")) {
+  presenceData.details = "Viewing 👤 profile";
+} else if (document.location.pathname.includes("/u/")) {
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:59]:
```suggestion
} else if (document.location.pathname.includes("/profile"))
  presenceData.details = "Viewing 👤 profile";
 else if (document.location.pathname.includes("/u/")) {
```

File: websites/D/DscJobs/presence.ts [hunk 6]
```diff
@@ -1,94 +1,114 @@
...
+  presenceData.state = `❤️ ${likes} & 👀 ${views}`;
+  presenceData.buttons = [
+    {
+      label: `View Resume`,
+      url: document.location.href
+    }
+  ];
+} else if (document.location.pathname.includes("/settings")) {
+  presenceData.details = `Editing 📜 curriculum vitae/resume`;
+} else if (document.location.pathname.includes("/legal")) {
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:95]:
```suggestion
} else if (document.location.pathname.includes("/settings"))
  presenceData.details = `Editing 📜 curriculum vitae/resume`;
  else if (document.location.pathname.includes("/legal")) {
```

File: websites/D/DscJobs/presence.ts [hunk 7]
```diff
@@ -1,94 +1,114 @@
...
+  const search = document.location.href.split("term=")[1].split("&")[0],
+    page = document.location.href.split("page=")[1].split("&")[0];
+  presenceData.details = `🔍 Searching for: ${search || "Nothing"}`;
+  presenceData.state = `📖 Page ${page}`;
+  presenceData.buttons = [
+    {
+      label: "View Results",
+      url: document.location.href
+    }
+  ];
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:25]:
```suggestion
```

File: websites/D/DscJobs/presence.ts [hunk 8]
```diff
@@ -1,94 +1,114 @@
...
+} else if (document.location.pathname.includes("/premium")) {
+  presenceData.details = "Viewing 💎 premium plan";
+  presenceData.buttons = [
+    {
+      label: "View Premium",
+      url: document.location.href
+    }
+  ];
```
Comment (ririxidev) [inline websites/D/DscJobs/presence.ts:48]:
```suggestion
  presenceData.buttons = [
    {
      label: "View Premium Plan",
      url: document.location.href
    }
  ];
```

**Review by ririxidev — CHANGES_REQUESTED** [review]
and run prettier over it

File: websites/D/DscJobs/presence.ts (hunk 4 above)
Comment (DiscordAnaxes) [reply websites/D/DscJobs/presence.ts:31]: I tried that, not sure how to add it with the   if (filters) presenceData.state = `💿 Filters: True`; already at state..

File: websites/D/DscJobs/dist/metadata.json (hunk 1 above)
Comment (DiscordAnaxes) [reply websites/D/DscJobs/dist/metadata.json:4]: Done 👍

File: websites/D/DscJobs/presence.ts (hunk 4 above)
Comment (SlowLife1661) [reply websites/D/DscJobs/presence.ts]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

Comment (DiscordAnaxes) [reply websites/D/DscJobs/presence.ts]: Oh true! Thanks. I have incorporated it.

**Review by ririxidev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**